### ISSUE TITLE
PHP 8.0 - Status 500 when viewing package list 

### ISSUE THREAD

**Issue by tinoest** [SimpleMachines/SMF#6305]
#### Description

Status 500 returned when viewing package list in php 8.0-rc1

### Steps to reproduce
1. Install php8.0-rc1
2. go to index.php?action=admin;area=packages
 

### Environment (complete as necessary)
- Version/Git revision: be238c1c45022e7a0e5977904f3caf6caba9d388
- Database Type: pgsql
- Database Version: 11.9
- PHP Version: 8.0-rc1

### Additional information/references

PHP Fatal error:  Uncaught Error: Unknown named parameter $type in Sources/Subs-List.php:132\nStack trace:\n#0 Sources/Packages.php(1499): createList()\n#1 Sources/Subs.php(5342): PackageBrowse()\n#2 Sources/Packages.php(91): call_helper()\n#3 Sources/Admin.php(499): Packages()\n#4 index.php(173): AdminMain()\n#5 {main}\n  thrown in Sources/Subs-List.php on line 132, referer: index.php

**Comment by jdarwood007** [comment]
Packages.php

```
				'params' => array('type' => $type),
```

```
				'params' => array($type),
```

**Comment by albertlast** [comment]
What is plan of supporting php?
Try to suppprt more and more do we cut on some places?

**Comment by jdarwood007** [comment]
No reason to cut any support at this point.

**Comment by albertlast** [comment]
@jdarwood007 should be this issue not closed by the pr #6311?

**Comment by jdarwood007** [comment]
Yes, I snuck the fix in for this in that PR.
